c7n/filters/core.py

```diff
@@ -335,6 +335,11 @@ def process_value_type(self, sentinel, value):
             # comparisons is intuitively wrong.
             return value, sentinel
 
+        elif self.vtype == 'cidr_size':
+            for i in value:
+                v = parse_cidrs(i)
+            return sentinel, v
+
```

Comment:
I'm fairly confident we can't do this as a generic filter... The core.py was built under the assumption that you are comparing a single-value sentinel with a single-value value. Line 254 in core.py I believe is the source of the problem since it only calls match() once and then the return value of `process_value_type` is piped directly to the operator() functions. Those functions only return a single bool, even if you pass in two lists. 

So again, I don't know how to proceed to make this actually act on a list of IPRanges that we want.